ARKit & Face Tracking

fand opened the pull request:
Added commands for ARKit and FaceTracking.
Command implementation is in private submodule.

nozomuplus commented on ZIGSIMPlus/Presenters/CommandDataSelectionPresenter.swift:
```diff
@@ -45,6 +45,12 @@ final class CommandDataSelectionPresenter: CommandDataSelectionPresenterProtocol
         }
         AppSettingModel.shared.isActiveByCommandData[label]?.toggle()
+
+        // Update commands availability
```
This comment should clarify the reason of running the same process as `init()`.
(Actually I didn't understand it.)

fand replied:
We need to update commandData every time users select/deselect commands, because "command.isAvailable" may change in some commands.
In this case, `Face Tracking` must be disabled when the user selects `ARKit`.

nozomuplus commented:
Good! Please check my comment.

fand commented:
I added a comment to explain why we have to update `commandDataToSelect`.
I also renamed `commandDatasToSelect` to `commandDataToSelectArray` because `datas` may sounds weird to English speakers.
